app/controllers/notifications_controller.rb

```diff
@@ -43,23 +43,17 @@ def convert_to_hash(string_obj)
     hash
   end
 
-  # rubocop:disable MethodLength
   def render_notification(notification)
     uniqueid = notification[:uniqueid]
     data = convert_to_hash(notification[:data])
-    if data[:type].include? 'comment'
-      comment_link(uniqueid, data)
-    elsif data[:type].include? 'accepted_ally_request'
-      accepted_ally_link(uniqueid, data)
-    elsif data[:type].include? 'new_ally_request'
-      new_ally_request_link(uniqueid, data)
-    elsif data[:type].include? 'group'
-      group_link(uniqueid, data)
-    elsif data[:type].include? 'meeting'
-      meeting_link(uniqueid, data)
+    case data[:type]
+    when /comment/ then comment_link(uniqueid, data)
+    when /accepted_ally_request/ then accepted_ally_link(uniqueid, data)
```

Comment:
Codeclimate seems to be complaining. Kindly check if the lines in red have been tested and add tests if they haven't